### PR TITLE
AHOYAPPS-961: Update to 4.0 SDK

### PR DESCRIPTION
**Pull request by timrozum**
https://issues.corp.twilio.com/browse/AHOYAPPS-961

1. Update to `TwilioVideo` 4.0.
1. Update CocoaPods version so fix [Snyk security issue](https://github.com/twilio/twilio-video-app-ios/pull/122).
1. Change deployment target to iOS 12.
1. Disable UI tests on CI because `TwilioVideo.VideoView` does not load in the VM environment.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -14,7 +16,7 @@ target 'Video-Internal' do
   pod 'FirebaseUI/Google', '~> 9'
   pod 'IGListDiffKit', '~> 4'
   pod 'KeychainAccess', '~> 4'
-  pod 'TwilioVideo', '~> 3'
+  pod 'TwilioVideo', '~> 4.0.0-rc15'
```
Comment (timrozum) [inline Podfile:19]: This will need to be changed to `4` before merge.

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This app is a sample video conferencing app that uses the [Twilio Programmable V
 
 iOS Deployment Target | Xcode Version | Swift Language Version
 ------------ | ------------- | -------------
-11.0 | 12.0 | Swift 5
+12.0 | 12.2 | Swift 5
```
Comment (paynerc) [inline README.md:21]: I think you can leave the Xcode requirement as 12.0

```suggestion
12.0 | 12.0 | Swift 5
```

File: Podfile (hunk 1 above)
Comment (paynerc) [reply Podfile:19]: Will approve but yes, please be sure to change this before merging.

**Review by paynerc — APPROVED** [review]
Approved pending the change to `~> 4.0`

**Review by paynerc — APPROVED** [review]
Let's do it!

**Pull request merged**